### PR TITLE
feat(java): sdk setup with android key

### PR DESCRIPTION
**Pull request by weakish**
see also leancloud/docs#3345

### CODE REVIEW

**Comment by leeyeh** [comment]
Deploy preview for *lc-docs-en* ready!

Built with commit 8b29ee04975919f89b33fe4aabf9cbee17e6f5af

https://deploy-preview-114--lc-docs-en.netlify.com

File: views/sdk_setup.tmpl [hunk 1]
```diff
@@ -512,6 +512,9 @@ Then specify the permissions needed by the SDK and declare the `MyLeanCloudApp`
   <!-- Feedback END -->
 </application>
 ```
+
+Since 6.1.0, we also support [initialize SDK without using appKey](sdk_setup_android_securely.html).
```
Comment (ghost) [inline views/sdk_setup.tmpl]: initialize -> initializing

File: views/sdk_setup_android_securely.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
+{% from "views/_data.njk" import libVersion as version %}
+
+# Use Android SDK without appKey
+
+Since 6.1.0, Leancloud Android SDK supports initialization without appKey,
```
Comment (ghost) [inline views/sdk_setup_android_securely.md]: Leancloud -> LeanCloud

File: views/sdk_setup_android_securely.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+}
+```
+
+Congratulations!
+You have just configured your application to initialize LeanCloud SDK without appKey.
+Now you can start developing your application as usual.
+
+## LeanEngine Runtime SDK
+
+If your android application will invoke [Cloud Function](leanengine_cloudfunction_guide-node.html),
```
Comment (ghost) [inline views/sdk_setup_android_securely.md]: android -> Android，后面还有一些

File: views/sdk_setup_android_securely.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+you need to make sure the LeanEngine runtime SDK you use supports this feature.
+
+Currently, the following LeanEngine runtime SDK versions support android initialization without appKey:
+
+- Python SDK：2.3.0 and later
```
Comment (ghost) [inline views/sdk_setup_android_securely.md]: ： -> :

File: views/sdk_setup_android_securely.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+#### Does Android SDK 6.1.0 still support the old way?
+
+Yes.
+If you prefer the old way, you can still use it,
+and you do not need to add native library to your project.
+
+#### Why the application crashes after switching to the new way?
+
+Probably because you do not add native library to your project.
```
Comment (ghost) [inline views/sdk_setup_android_securely.md]: do -> did

File: views/sdk_setup_android_securely.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+Please refer to section [JNI Native Library](#jni-native-library) above.
+
+#### Why all requests return `{"code":401,"error":"Unauthorized."}` error?
+
+Probably because your apk does not properly configure signing.
```
Comment (ghost) [inline views/sdk_setup_android_securely.md]: does not properly configure signing -> did not have signing properly configured

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**